### PR TITLE
Edit PPP

### PR DESCRIPTION
**Pull request by yucheng11122017**

### CODE REVIEW

**Review by leezhixuan — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -296,13 +296,14 @@ Step 3. A `FindPredicate`  which will only return `true` if contact's name conta
 
 Step 4. This`FindPredicate` is passed into `ModelManager#updateFilteredPersonList`, updating the filtered list.
 
-Step 5. CONNECTIONS' `UI` observes the filtered list is updated and displayed the updated filtered list in `PersonListPanel`.
+Step 5. CONNECTIONS' `UI` observes the filtered list is updated and displayed the updated filtered list in `PersonListPanel` which will only contain contacts whose name contains `David` **while also having** `friend` **and**
+`football` tagged to them.
```
Comment (Justinhoejj) [inline docs/DeveloperGuide.md:300]: Phrasing is a bit odd here not too sure what the sentence is trying to say.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -296,13 +296,14 @@ Step 3. A `FindPredicate`  which will only return `true` if contact's name conta
...
 
 #### Design considerations:
 
 **Aspect: How Find executes:**
 
-* **Option 1:** Utilise `NameContainsKeywordsPredicate` and `PersonsTagsContainsCaseInsensitiveTags`
+* **Option 1:** Utilise two predicates, one to check if name contains the keywords and one to check if tags contains the keywords
```
Comment (Justinhoejj) [inline docs/DeveloperGuide.md:306]: Missing fullstop.

**Review by Justinhoejj — APPROVED** [review]
Nice code clean up! Small issues.

**Pull request merged**